docs/contributing/introduction-to-contributing.md

```diff
@@ -52,7 +52,7 @@ See [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) for m
 
 ## Pull Requests (PR)
 
-**_We actively welcome your pull requests. However, you must link your work to an existing issue._**
+_We actively welcome your pull requests. However, you must ensure that **you are assigned** to an existing issue before working on changes, and you need to **link your work to the issue** in your PR form._
```

Comment:
We want to include the horizontal lines here just as the requirement in the issue. And please remove the underscore.

 ```suggestion
---

We actively welcome your pull requests. However, you must ensure that **you are assigned** to an existing issue before working on changes, and you need to **link your work to the issue** in your PR form.

---
```